pep-0698.rst

```diff
@@ -0,0 +1,439 @@
+PEP: 698
+Title: Override Decorator for Static Typing
+Author: Steven Troxler <steven.troxler@gmail.com>,
+        Joshua Xu <jxu425 at fb.com>,
+        Shannon Zhu <szhu@fb.com>
+Status: Draft
+Sponsor: Jelle Zijlstra <jelle.zijlstra at gmail.com>
```

Comment:
```suggestion
Sponsor: Jelle Zijlstra <jelle.zijlstra at gmail.com>
```

FYI, you don't need to manually obfuscate your email address here; this actually impedes the automatic obfuscator that uses a bunch of Unicode characters to actually make it much less trivial for spambots to detect the address

Reply:
Good to know! I can probably change Josh right now

I think right now the linter actually complains if I use a de-obfuscated email for Jelle; it doesn't allow two versions of the same email and existing PEPs have it this way.


I could make a separate commit (against the whole peps repo) to change all instances for Jelle

Reply:
> I think right now the linter actually complains if I use a de-obfuscated email for Jelle; it doesn't allow two versions of the same email and existing PEPs have it this way.

FWIW, its actually the PEP header transform Sphinx extension in the PEP build system, rather than the linter—at one point I was working on handling this, but its not as trivial as it should be without unifying the header checks and processing and conforming old email addresses in the legacy bespoke parenthesis format to the standard RFC 2822 / 5322 format.

> I could make a separate commit (against the whole peps repo) to change all instances for Jelle

IMO, that would be best as a separate PR, assuming Jelle agrees to the change—best to leave it as is for now, I guess